Sources/SimpleSalesforceSDK/Models/Errors.swift

```diff
@@ -7,6 +7,7 @@ public enum SSSDKError: Error {
   case authNoRefreshTokenError
   case authRefreshFailedError
   case authIntrospectFailedError
+  case invalidUrlError
```

Comment:
maybe this should be 
```suggestion
  case invalidUrlError(url: String)
```

That way you can include the url in the error for the sdk user to inspect?